rllib/evaluation/rollout_worker.py

```diff
@@ -1861,6 +1880,20 @@ def _build_policy_map(
             if connectors_enabled and name in self.policy_map:
                 create_connectors_for_policy(self.policy_map[name], policy_config)
 
+                # As long as the historic filter synchronization mechanism is in
+                # place, we need to put filters into self.filters so that they get
+                # synchronized
+                filter_connectors = self.policy_map[name].agent_connectors[
```

Comment:
will it be worth it to create a small local util function for this logic, so we don't duplicate it twice here and above?
also it will make the high level code look nicer:
```
if connectors_enabled:
    policy = self.policy_map[name]
    create_connectors_for_policy(policy)
    maybe_get_filters_for_syncing(policy)
```
hopefully the long-ish if-else block above will be simplified too.

Reply:
Done